ct-app/core/components/conversions.py

```
@@ -0,0 +1,33 @@
+from typing import Any
+
+from core.components.balance import Balance
+
+
+def convert_unit(value: Any):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)